web/src/constants/index.ts

```diff
@@ -1,27 +1,32 @@
+import { CanCan } from 'src/containers';
+import { AbilitiesInterface } from 'src/modules';
 import { DropdownElem } from '../components/Order';
 
 export const PG_TITLE_PREFIX = 'Cryptobase';
 
-export const pgRoutes = (isLoggedIn: boolean, isLight?: boolean): string[][] => {
+export const pgRoutes = (isLoggedIn: boolean, abilities: AbilitiesInterface, isLight?: boolean): string[][] => {
     const routes = [
         ['page.header.navbar.trade', '/trading/', `trade${isLight ? 'Light' : ''}`],
-        ['page.header.navbar.quick.exchange', '/quick-exchange', `quick_exchange${isLight ? 'Light' : ''}`],
+        CanCan.checkAbilityByAction('read', 'QuickExchange', abilities) && ['page.header.navbar.quick.exchange', '/quick-exchange', `quick_exchange${isLight ? 'Light' : ''}`],
         ['page.header.navbar.wallets', '/wallets', `wallets${isLight ? 'Light' : ''}`],
         ['page.header.navbar.openOrders', '/orders', `orders${isLight ? 'Light' : ''}`],
         ['page.header.navbar.history', '/history', `history${isLight ? 'Light' : ''}`],
-        ['page.header.navbar.api', '/docs', `api${isLight ? 'Light' : ''}`],
+        CanCan.checkAbilityByAction('read', 'P2P', abilities) && ['page.header.navbar.p2p', '/p2p', `p2p${isLight ? 'Light' : ''}`],
+        ['page.header.navbar.api', '/api', `api${isLight ? 'Light' : ''}`],
```

Comment:
check please if /api exists on Baseapp

Reply:
Changed to /docs as /api is not exists, thanks.